xhr.bs

```diff
@@ -825,7 +809,8 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
 
     <p class=note>These steps are only invoked when new bytes are transmitted.
 
-    <p>To <a>process request end-of-body</a> for <var>request</var>, run these steps:
+   <li>
+    <p>Let <var>processRequestEndOfBody</var>, given a <var>request</var>, be these steps:
 
     <ol>
      <li><p>Set <a>this</a>'s <a>upload complete flag</a>.
```

Comment:
When this runs, this is undefined. Does this mean we basically have to do that = this before these kind of things? @domenic 

This is an existing issue that I think should be addressed in a follow-up, but wanted to raise it here as I noticed it when writing this.

Reply:
I've been treating spec steps as "arrow functions" which close over "this". "this" is unique for the entire definition of the method, so this doesn't seem likely to cause confusion.

Reply:
I see, I guess we can keep this as-is then. Thanks!